### PR TITLE
`Development`: Add assessment documentation

### PR DESCRIPTION
**Pull request by Hialus**
#### General

- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

Documentation for the assessment of exercises is currently missing.
See [the confluence page](https://confluence.ase.in.tum.de/display/ArTEMiS/Feature+documentation).

### Description

Add documentation page for assessment.

### Steps for Testing

1. Read the documentation at https://artemis-platform--5483.org.readthedocs.build/en/5483/user/exercises/assessment/

#### Documentation Review
- [x] Review 1
- [x] Review 2

### CODE REVIEW

**Review by jschmautzer — APPROVED** [review]
looks good and descriptive :)

**Review by luisa-we — APPROVED** [review]
Good documentation, very detailed!

**Review by MetinLamby — DISMISSED** [review]
nicely written!

File: docs/user/exercises/assessment.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+
+Overview
+--------
+
+Artemis offers three different modes of exercise assessment:
+
+- **Manual:** Tutors must manually grade the submissions of students.
+- **Automatic:** Artemis automatically grades the submissions of students.
+- **Semi-Automatic:** Artemis automatically grades students' submissions, but the tutors manually grade all submissions afterward.
```
Comment (Marvin-Vitcu) [inline docs/user/exercises/assessment.rst:18]: Be consistent with the previous definitions:

```suggestion
- **Semi-Automatic:** Artemis automatically grades the submissions of students, but the tutors manually grade all submissions afterward.
```

**Review by Marvin-Vitcu — COMMENTED** [review]
Tested on TS1. The link for Assessment on the "About" page is missing.

File: docs/user/exercises/assessment.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+-----------------
+
+Manual assessment is available for all exercise types except quiz exercises.
+The manual assessment begins after the deadline of an exercise has passed for all students and is double-blind. This means that the tutors do not know the names of the students they assess, and the students do not know the identity of the tutors.
+
+Instructors can use the assessment training process to make the grading more consistent. The assessment training process is a series of example submissions and assessments defined by the instructor, which the tutors must first read through.
+
+Students have the option to rate the assessments of the tutors. They can also complain or ask for more feedback.
+
+Assessment dashboard
```
Comment (Strohgelaender) [inline docs/user/exercises/assessment.rst:30]: Also change the other headings.
```suggestion
Assessment Dashboard
```

File: docs/user/exercises/assessment.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+^^^^^^^^^^^^^^^^^^^^
+
+To keep track of the manual assessments, Artemis offers the assessment dashboard.
+The assessment dashboard shows the assessment progress of each exercise by showing the state of the exercise, the total number of submissions, the number of submissions that have been assessed, and the number of complaints and more feedback requests.
```
Comment (Strohgelaender) [inline docs/user/exercises/assessment.rst:34]:
```suggestion
It shows the assessment progress of each exercise by showing the state of the exercise, the total number of submissions, the number of submissions that have been assessed, and the number of complaints and more feedback requests.
```

File: docs/user/exercises/assessment.rst [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+Complaints
+----------
+
+After receiving an assessment, students can complain once about the assessment of an exercise if the instructor enabled this option and the students think the assessment is erroneous.
+The student has to write an additional text when submitting a complaint to justify the reevaluation.
+    .. figure:: assessment/complaint.png
+            :alt: The student complains about the assessment of an exercise
+            :align: center
+
+A complaint leads to a reevaluation of the submission by another tutor. This tutor sees the existing submission and the complaint reason. The tutor can then either accept the complaint or reject it.
```
Comment (Strohgelaender) [inline docs/user/exercises/assessment.rst:66]:
```suggestion
A complaint leads to a reevaluation of the submission by another tutor. This tutor sees the existing assessment and the complaint reason. The tutor can then either accept or reject the complaint.
```

File: docs/user/exercises/assessment.rst [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+If the tutor accepts the complaint, they can modify the assessment's score.
+If the tutor rejects the complaint, they cannot do this.
```
Comment (Strohgelaender) [inline docs/user/exercises/assessment.rst:68]:
```suggestion
Only if the tutor accepts the complaint, they can modify the assessment's score.
```

**Review by Strohgelaender — CHANGES_REQUESTED** [review]
Also add the link to the german translation.

File: src/main/webapp/i18n/en/aboutUs.json [hunk 6]
```diff
@@ -26,7 +26,7 @@
             "text": "<a href={{ gradingUrl }}>Grading:</a> Calculate and display grades using configurable grade keys for courses and exams. Export as csv files to university systems (e.g. CAMPUSonline)."
         },
         "assessment": {
-            "text": "<b>Assessment:</b> Double-blind grading with structured criteria, integrated training, a leader board, ratings, and complaints improve consistency and fairness."
+            "text": "<a href={{ assessmentUrl }}>Assessmen:t</a> Double-blind grading with structured criteria, integrated training, a leader board, ratings, and complaints improve consistency and fairness."
```
Comment (Strohgelaender) [inline src/main/webapp/i18n/en/aboutUs.json:29]:
```suggestion
            "text": "<a href={{ assessmentUrl }}>Assessment:</a> Double-blind grading with structured criteria, integrated training, a leader board, ratings, and complaints improve consistency and fairness."
```

Comment (Hialus) [reply src/main/webapp/i18n/en/aboutUs.json:29]: Whoops 😅

File: docs/user/exercises/assessment.rst [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+Overview
+--------
+
+Artemis offers three different modes of exercise assessment:
+
+- **Manual:** Tutors must manually grade the submissions of students.
+- **Automatic:** Artemis automatically grades the submissions of students.
+- **Semi-Automatic:** Artemis automatically grades the submissions of students, but the tutors manually grade all submissions afterward.
+
+Manual assessment
```
Comment (Strohgelaender) [inline docs/user/exercises/assessment.rst:20]:
```suggestion
Manual Assessment
```

File: docs/user/exercises/assessment.rst [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+        :alt: A picture of the assessment dashboard (of an exam)
+        :align: center
+
+Artemis also offers a way for instructors to monitor the tutors' assessments. The first part of this is the tutor leaderboard, which is visible to all tutors. The tutor leaderboard shows the number of assessments each tutor has done and the amount of more feedback requests and accepted complaints about them.
+It also shows the average score the tutor has given and the average rating they received for their assessments.
+
+Each exercise also has its own assessment dashboard that shows all of this information for a single exercise.
+
+
+Automatic assessment
```
Comment (Strohgelaender) [inline docs/user/exercises/assessment.rst:47]:
```suggestion
Automatic Assessment
```

File: docs/user/exercises/assessment.rst [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+
+Overview
+--------
+
+Artemis offers three different modes of exercise assessment:
+
+- **Manual:** Tutors must manually grade the submissions of students.
+- **Automatic:** Artemis automatically grades the submissions of students.
+- **Semi-Automatic:** Artemis automatically grades the submissions of students, but the tutors manually grade all submissions afterward.
```
Comment (Strohgelaender) [inline docs/user/exercises/assessment.rst:18]: Not sure about the exact formulation, but I think this sentence should be more clear. 
```suggestion
- **Semi-Automatic:** Artemis provides an automatic starting point for the tutors to manually improve the grading afterward.
```

**Review by nairaarnez — APPROVED** [review]
It looks good! I just have two small suggestions: I would enumerate 
"by showing: 
1. The state of the exercise
2. The total number of submissions
3. etc"
In the text and the screenshot 
image

And I would add a screenshot of the view described here:
image

Just suggestions, approved!

**Review by alaraozdenler — APPROVED** [review]
Documentation looks good, no errors found

**Review by luisa-we — APPROVED** [review]
Documentation still looks good!
Tested on TS1, link is there.

**Review by manuelmanso — APPROVED** [review]
Docs are clear and well organized!

**Pull request merged**